### PR TITLE
feat: sort collection activity

### PR DESCRIPTION
**Pull request by yuesth**
- sort by price, from, to

### CODE REVIEW

File: components/Collection/CollectionActivity.js [hunk 1]
```diff
@@ -93,7 +160,136 @@ const CollectionActivity = ({ activities, fetchData, hasMore, dailyVolume }) =>
...
+																			pathname: `/collection/${collectionId}`,
+																			query: {
+																				headerActivities: d.id,
+																				sortActivities: 'asc',
+																				tab: `activity`,
+																			},
+																		},
+																		{ shallow: true, scroll: false }
+																	)
+																}}
```
Comment (ahnafalfariza) [inline components/Collection/CollectionActivity.js:193]: untuk `router.push` second argumentsnya apakah perlu? soalnya kalo cek di [docsnya](https://nextjs.org/docs/api-reference/next/router), itukan optional decorator ya, (untuk di show di browser url bar). jadi sepertinya bisa di first argument aja. jadi mungkin bisa gini aja
```js
router.push(
  {
    pathname: `/collection/${collectionId}`,
    query: {
      headerActivities: d.id,
      sortActivities: 'asc',
      tab: `activity`,
    },
  },
  '/collection/${collectionId}'
  { shallow: true, scroll: false }
)
```
wdyt?

File: pages/collection/[collection_id].js [hunk 2]
```diff
@@ -189,12 +189,25 @@ const CollectionPage = ({ collectionId, collection, serverQuery }) => {
...
+				  }
+				: {
+						collection_id: collectionId,
+						filter: 'sale',
+						__skip: _page * LIMIT_ACTIVITY,
+						__limit: LIMIT_ACTIVITY,
+						__sort: `${headerActivities}${sortActivities === `asc` ? `::1` : ``}${
+							sortActivities === `desc` ? `::-1` : ``
+						}`,
+				  }
```
Comment (ahnafalfariza) [inline pages/collection/[collection_id].js:210]: bagian sortActivitiesnya langsung gini, how?
```js
{sortActivities === `asc` ? `::1` : `::-1`}
```

File: components/Collection/CollectionActivity.js [hunk 3]
```diff
@@ -93,7 +160,136 @@ const CollectionActivity = ({ activities, fetchData, hasMore, dailyVolume }) =>
...
+																		{
+																			pathname: `/collection/${collectionId}`,
+																			query: {
+																				headerActivities: d.id,
+																				sortActivities: 'desc',
+																				tab: `activity`,
+																			},
+																		},
+																		{ shallow: true, scroll: false }
+																	)
```
Comment (ahnafalfariza) [inline components/Collection/CollectionActivity.js:220]: same as above

File: components/Collection/CollectionActivity.js [hunk 4]
```diff
@@ -93,7 +160,136 @@ const CollectionActivity = ({ activities, fetchData, hasMore, dailyVolume }) =>
...
+																			pathname: `/collection/${collectionId}`,
+																			query: {
+																				headerActivities: d.id,
+																				sortActivities: '',
+																				tab: `activity`,
+																			},
+																		},
+																		{ shallow: true, scroll: false }
+																	)
+																}}
```
Comment (ahnafalfariza) [inline components/Collection/CollectionActivity.js:253]: same as above

File: components/Collection/CollectionActivity.js [hunk 5]
```diff
@@ -93,7 +160,136 @@ const CollectionActivity = ({ activities, fetchData, hasMore, dailyVolume }) =>
...
+																	pathname: `/collection/${collectionId}`,
+																	query: {
+																		headerActivities: d.id,
+																		sortActivities: 'asc',
+																		tab: `activity`,
+																	},
+																},
+																{ shallow: true, scroll: false }
+															)
+														}}
```
Comment (ahnafalfariza) [inline components/Collection/CollectionActivity.js:286]: same as above

**Review by ahnafalfariza — CHANGES_REQUESTED** [review]

File: components/Collection/CollectionActivity.js (hunk 1 above)
Comment (yuesth) [reply components/Collection/CollectionActivity.js:193]: woke gan, untuk query `?tab=activity` nya perlu ditambahin gan di 2nd argumentnya?

File: pages/collection/[collection_id].js (hunk 2 above)
Comment (yuesth) [reply pages/collection/[collection_id].js:210]: woke fixing gan

File: components/Collection/CollectionActivity.js (hunk 3 above)
Comment (yuesth) [reply components/Collection/CollectionActivity.js:220]: fixing gan

File: components/Collection/CollectionActivity.js (hunk 4 above)
Comment (yuesth) [reply components/Collection/CollectionActivity.js:253]: fixing gan

File: components/Collection/CollectionActivity.js (hunk 5 above)
Comment (yuesth) [reply components/Collection/CollectionActivity.js:286]: fixing gan

**Review by ahnafalfariza — APPROVED** [review]

**Comment by yuesth** [comment]
close #430

**Pull request merged**